client/ayon_core/hosts/hiero/plugins/create/create_shot_clip.py

```diff
@@ -1,247 +1,233 @@
-                    "toolTip": "Process subsets with corresponding audio",  # noqa
-                    "order": 3},
-                "sourceResolution": {
-                    "value": False,
-                    "type": "QCheckBox",
-                    "label": "Source resolution",
-                    "target": "tag",
-                    "toolTip": "Is resloution taken from timeline or source?",  # noqa
-                    "order": 4},
-            }
-        },
-        "frameRangeAttr": {
-            "type": "section",
-            "label": "Shot Attributes",
-            "target": "ui",
-            "order": 4,
-            "value": {
-                "workfileFrameStart": {
-                    "value": 1001,
-                    "type": "QSpinBox",
-                    "label": "Workfiles Start Frame",
-                    "target": "tag",
-                    "toolTip": "Set workfile starting frame number",  # noqa
-                    "order": 0
-                },
-                "handleStart": {
-                    "value": 0,
-                    "type": "QSpinBox",
-                    "label": "Handle Start",
-                    "target": "tag",
-                    "toolTip": "Handle at start of clip",  # noqa
-                    "order": 1
-                },
-                "handleEnd": {
-                    "value": 0,
-                    "type": "QSpinBox",
-                    "label": "Handle End",
-                    "target": "tag",
-                    "toolTip": "Handle at end of clip",  # noqa
-                    "order": 2
-                }
-            }
-        }
-    }
+    create_allow_context_change = False
+    create_allow_thumbnail = False
+
+    def get_pre_create_attr_defs(self):
+
+        def header_label(text):
+            return f"<br><b>{text}</b>"
+
+        tokens_help = """\nUsable tokens:
+    {_clip_}: name of used clip
+    {_track_}: name of parent track layer
+    {_sequence_}: name of parent sequence (timeline)"""
+        # gui_name = "OpenPype publish attributes creator"
+        # gui_info = "Define sequential rename and fill hierarchy data."
+        gui_tracks = [track.name()
+            for track in lib.get_current_sequence().videoTracks()]
+
+        # Project settings might be applied to this creator via
+        # the inherited `Creator.apply_settings`
+        presets = self.presets
+
+        return [
+
+            BoolDef("use_selection",
+                    label="Use only clips with <b>Chocolate</b>  clip color",
+                    tooltip=(
+                        "When enabled only clips of Chocolate clip color are "
+                        "considered.\n\n"
+                        "Acts as a replacement to 'Use selection' because "
+                        "Resolves API exposes no functionality to retrieve "
+                        "the currently selected timeline items."
+                    ),
+                    default=True),
+
```

Comment:
In a Resolve workflow, this is the only way it makes sense. In Hiero, you can just pick from the selected clips. We should add an option to switch between applying to selected clips or not. [Here it can be taken from](https://github.com/ynput/ayon-core/blob/5e77ba5619caa455a4bef0b1fb932a03a9db3e23/client/ayon_core/hosts/nuke/plugins/create/create_write_image.py#L37)